grid2op/tests/test_alert.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+import warnings
+import numpy as np
+import unittest
+import os
+import tempfile
+from grid2op.tests.helper_path_test import *
+
+from grid2op.operator_attention import LinearAttentionBudget
+from grid2op import make
+from grid2op.Reward import RedispReward, _AlarmScore
+from grid2op.Exceptions import Grid2OpException
+from grid2op.Runner import Runner
+from grid2op.Environment import Environment
+from grid2op.Episode import EpisodeData
+
+class TestAlert(unittest.TestCase):
+    """test the basic bahavior of the assistant alert feature"""
+
+    def test_init_default_param(self) -> None : 
+        self.env_nm = os.path.join(
+            PATH_DATA_TEST, "l2rpn_idf_2023_with_alert"
+        )
+        self.env = make(self.env_nm, test=True)
```

Comment:
can you add the `with warnings.catch_warnings() etc.` idioms that is present everywhere in the test when making an environment please ?